### PR TITLE
Blog: Our Journey as Outreachy Interns with Bioconductor

### PR DESCRIPTION
**Pull request by sonali8434**

### CODE REVIEW

File: posts/2023-06-29-OutreachyInternshipExperience/index.qmd
Comment (jwokaty) [inline posts/2023-06-29-OutreachyInternshipExperience/index.qmd:94]: I know you're still probably editing, but I wanted to mention a few things.

I want to suggest moving the personal introduction before the project you're working because you are the "news" of this blog post. Your introduction is great. You tell us who you are, where you're at, what you're doing now, your interests, etc. 

Rather than a large separate section about the project, it might be better to write a shorter introduction/explanation you're briefly explaining it to someone who doesn't know anything about it. The second paragraph in your personal section is a good opportunity to introduce the project. Then you can remove the large, separate project section.

Also, in that same paragraph, when you wrote about "various projects available", I think you must be referring to the Outreachy contribution period, but I think most Bioconductor community members don't know the process, so you might want to explain it. 

It might also be good to talk more concretely your contributions to the project, for example, converting vignettes, doing outreach with maintainers, trying to improve the conversion process, etc.

You can sing your own praises here because you are doing good work :)

**Review by jwokaty — COMMENTED** [review]
I just left some initial feedback as I know you're still working on this.

File: posts/2023-06-29-OutreachyInternshipExperience/index.qmd
Comment (sonali8434) [reply posts/2023-06-29-OutreachyInternshipExperience/index.qmd]: I have made some changes after the first review kindly review the changes.

File: posts/2023-06-29-OutreachyInternshipExperience/index.qmd
Comment (jwokaty) [inline posts/2023-06-29-OutreachyInternshipExperience/index.qmd:18]: How about replacing 

```
During the application period for the May 2023 internship round,
Bioconductor opened its doors to the Outreachy Internship program.
```

with 

```
Three interns and three Bioconductor community projects are participating in the May 2023 <a href="outreachy.org">Outreachy</a> internship round.
```

File: posts/2023-06-29-OutreachyInternshipExperience/index.qmd
Comment (jwokaty) [inline posts/2023-06-29-OutreachyInternshipExperience/index.qmd:51]: Could you link to the sweave2rmd repo?

**Review by jwokaty — CHANGES_REQUESTED** [review]
I like this revision. The specific achievements give someone a good understanding of what your work entails.

I think we should probably remove the section 20-28 as it's very similar to the previous intern blog post.

File: posts/2023-06-29-OutreachyInternshipExperience/index.qmd
Comment (sonali8434) [reply posts/2023-06-29-OutreachyInternshipExperience/index.qmd:28]: I will rewrite this.

File: posts/2023-06-29-OutreachyInternshipExperience/index.qmd
Comment (sonali8434) [reply posts/2023-06-29-OutreachyInternshipExperience/index.qmd]: done

**Review by jwokaty — APPROVED** [review]

File: posts/2023-06-29-OutreachyInternshipExperience/index.qmd
Comment (jwokaty) [inline posts/2023-06-29-OutreachyInternshipExperience/index.qmd:120]: Fixing punctuation and capitalization:

    to optimization of unit tests and examples, current task include improving 
    documentation in packages, creating an example microbiome data science 
    workflow from database to results, optimization of certain methods and 
    testing, issue reporting, bug fixing.

File: posts/2023-06-29-OutreachyInternshipExperience/index.qmd
Comment (jwokaty) [inline posts/2023-06-29-OutreachyInternshipExperience/index.qmd:84]: I think rather than having a separate section about the project, it's better to include the paragraph for 109-120 here then you can remove the separate section about the project. We want a brief explanation of the project because this will be a long blog post.

**Review by jwokaty — CHANGES_REQUESTED** [review]

File: posts/2023-06-29-OutreachyInternshipExperience/index.qmd
Comment (jwokaty) [inline posts/2023-06-29-OutreachyInternshipExperience/index.qmd:79]: Is the project called Microbiome Project or the Miaverse Project?

File: posts/2023-06-29-OutreachyInternshipExperience/index.qmd
Comment (jwokaty) [inline posts/2023-06-29-OutreachyInternshipExperience/index.qmd:107]: @Daenarys8  We should follow the capitalization that the miaverse uses, which I think is always lowercase looking at the website.

File: posts/2023-06-29-OutreachyInternshipExperience/index.qmd
Comment (jwokaty) [inline posts/2023-06-29-OutreachyInternshipExperience/index.qmd:87]: @Daenarys8 Please include a link to the miaverse website.

File: posts/2023-06-29-OutreachyInternshipExperience/index.qmd
Comment (jwokaty) [inline posts/2023-06-29-OutreachyInternshipExperience/index.qmd:139]: @Daenarys8 Please replace this section with a section about how readers can read your blog. See Sonali's section for how she did it. You could also just remove this section and add a sentence to your last paragraph to tell readers to check out your blog.

File: posts/2023-06-29-OutreachyInternshipExperience/index.qmd
Comment (jwokaty) [inline posts/2023-06-29-OutreachyInternshipExperience/index.qmd:169]: @atrayees Not all of the Bioconductor community is familiar with Outreachy so the project names may be unfamiliar to a lot of readers, so I suggested to the other interns to use project names that might be familiar within the community. You can say, for example, the `"Microbiome Study Curation" project with <a href="bugsigdb.org">BugSigDb</a>`. Note, we use the project name and link in case anyone is interested in learning more.

File: posts/2023-06-29-OutreachyInternshipExperience/index.qmd
Comment (jwokaty) [inline posts/2023-06-29-OutreachyInternshipExperience/index.qmd:180]: @atrayees Also, the community is not very familiar with the Outreachy application process, including the contribution period It might be helpful to explain it a bit for the readers. For example

    The [Outreachy contribution period](https://www.outreachy.org/docs/applicant/#make-contributions) was one of the best experiences I have ever had as a student.

File: posts/2023-06-29-OutreachyInternshipExperience/index.qmd
Comment (jwokaty) [inline posts/2023-06-29-OutreachyInternshipExperience/index.qmd:191]: @atrayees Can you please add a sentence or two about how readers can follow your progress by reading your blog? See how Sonali wrote it in her section as an example.

**Review by jwokaty — CHANGES_REQUESTED** [review]
@Daenarys8 @atrayees Thank you so much for adding your sections. It's really great to learn about your experiences and I'm excited that we'll be able to share this with the community soon.

Please keep in mind that the Bioconductor community is still learning about Outreachy and may not be familiar with some aspects of the application process. It's my hope that we can bridge this information gap, share the work you're doing in our community, and get more mentors to submit their projects and have more spots for future interns. I tried to provide some guidance to help you to help the community understand better. I also want to remind you to include links to your projects and blogs.

@Daenarys8 @atrayees @sonali8434 Since there are three of you working together, please make sure that the blog post "flows" from one section to the next.

I also want to ask you to add a section at the end mentioning that the application opens in August for Outreachy applicants and that mentors can start submitting projects in August. Please include the links. And finally, there will be an Outreachy Birds of a Feather at Bioc2023:https://bioc2023.bioconductor.org/abstracts/track12/.

File: posts/2023-06-29-OutreachyInternshipExperience/index.qmd
Comment (atrayees) [reply posts/2023-06-29-OutreachyInternshipExperience/index.qmd]: I have added a couple of sentences as well as the link to my blog, hope it looks better. Thanks a lot for the review!

File: posts/2023-06-29-OutreachyInternshipExperience/index.qmd
Comment (atrayees) [reply posts/2023-06-29-OutreachyInternshipExperience/index.qmd]: Thank you, that make sense! I have made the changes

File: posts/2023-06-29-OutreachyInternshipExperience/index.qmd
Comment (atrayees) [reply posts/2023-06-29-OutreachyInternshipExperience/index.qmd]: Thanks a lot, I committed the changes.

**Comment by sonali8434** [comment]
Hello @jwokaty, we have added the introduction and the last part of the blog as you suggested. Please review these sections. Now, the blog is complete from our side. If you have any suggestions, please let us know.

Thank you!

File: posts/2023-06-29-OutreachyInternshipExperience/index.qmd
Comment (jwokaty) [inline posts/2023-06-29-OutreachyInternshipExperience/index.qmd:24]: I just want clarify the relationship between Outreachy and Bioconductor a little more with

`Outreachy is one such project that partners with open source and open science communities to provide internships to underrepresented`

File: posts/2023-06-29-OutreachyInternshipExperience/index.qmd
Comment (jwokaty) [inline posts/2023-06-29-OutreachyInternshipExperience/index.qmd:49]: Can we put the links in R Markdown format? (I sometimes forget too.)

    [Outreachy](outreachy.org)

File: posts/2023-06-29-OutreachyInternshipExperience/index.qmd
Comment (jwokaty) [inline posts/2023-06-29-OutreachyInternshipExperience/index.qmd:78]: Let's use R Markdown format:

    [Sweave2Rmd](https://github.com/Bioconductor/sweave2rmd)

File: posts/2023-06-29-OutreachyInternshipExperience/index.qmd
Comment (jwokaty) [inline posts/2023-06-29-OutreachyInternshipExperience/index.qmd:116]: Let's use R Markdown format:

    [miaverse](https://microbiome.github.io)

File: posts/2023-06-29-OutreachyInternshipExperience/index.qmd
Comment (jwokaty) [inline posts/2023-06-29-OutreachyInternshipExperience/index.qmd:152]: Let's use R Markdown format:

    [daenarys.dreamwidth.org](daenarys.dreamwidth.org)

File: posts/2023-06-29-OutreachyInternshipExperience/index.qmd
Comment (jwokaty) [inline posts/2023-06-29-OutreachyInternshipExperience/index.qmd:226]: Replace 'of' with 'for'

    December 2023 cohort can [apply](https://www.outreachy.org/apply/) for the

File: posts/2023-06-29-OutreachyInternshipExperience/index.qmd
Comment (jwokaty) [inline posts/2023-06-29-OutreachyInternshipExperience/index.qmd:234]: I'm still a mentor, so maybe you can replace `previous mentor and community coordinator` with `Bioconductor community coordinator for Outreachy`

**Review by jwokaty — CHANGES_REQUESTED** [review]
This is great! just a few small wording changes and formatting changes to make sure we're using R Markdown format. This should be ready to merge after these changes are complete.

File: posts/2023-06-29-OutreachyInternshipExperience/index.qmd
Comment (sonali8434) [reply posts/2023-06-29-OutreachyInternshipExperience/index.qmd]: Done

**Comment by sonali8434** [comment]
@jwokaty Done all the suggested changes. Please Review.

**Review by jwokaty — APPROVED** [review]
@sonali8434 Thank you! And thanks again everyone for sharing your experience with the community :)

**Comment by mblue9** [comment]
Sorry for delay, I will share this on social media shortly.

**Comment by mblue9** [comment]
@jwokaty before I share it I'm going to make a PR to adjust the photo sizes as one is huge and the other small. Noticed there's no photo of @Daenarys8 is there one we could add?

**Pull request merged**
